src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -498,7 +498,7 @@ private ArgumentsBuilder createBaselineArgumentsForFuncotator(final String varia
     // DO NOT ADD THIS TO ANY TEST GROUPS!
```

Comment:
You still need to add some kind of `Funcotator` test to cover the fix for dbsnp. Anything reasonable that fails without the fix and passes with it is fine, provided that it covers the actual files published in the latest datasource release. How about a cloud test (`@Test(groups={"cloud"})`) that does 4 small queries on the actual hosted dbsnp VCF from the latest Funcotator datasources in GCS: a tiny interval on each of chr1, chrMT, chrX, and chrY. The test could just assert that you get a certain expected number of records from each query. You can do this trivially using a `FeatureDataSource`.

Reply:
Sounds good to me.